spec/models/carriers/usps_calculator_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'spec_helper'
+
+module ActiveShipping
+  describe Spree::Calculator::Shipping::Usps do
+    WebMock.disable_net_connect!
+
+    let(:address) { FactoryGirl.create(:address) }
+    let(:stock_location) { FactoryGirl.create(:stock_location) }
+    let!(:order) do
+      order = FactoryGirl.create(:order_with_line_items, :ship_address => address, :line_items_count => 2)
+      order.line_items.first.tap do |line_item|
+        line_item.quantity = 2
+        line_item.variant.save
+        line_item.variant.weight = 1
+        line_item.variant.save
+        line_item.save
+        # product packages?
+      end
+      order.line_items.last.tap do |line_item|
+        line_item.quantity = 2
+        line_item.variant.save
+        line_item.variant.weight = 2
+        line_item.variant.save
+        line_item.save
+        # product packages?
+      end
+      order
+    end
+
+    let(:carrier) { ActiveShipping::USPS.new(:login => "FAKEFAKEFAKE") }
+    let(:calculator) { Spree::Calculator::Shipping::Usps::ExpressMail.new }
+    let(:response) { double('response', :rates => rates, :params => {}) }
+    let(:package) { order.shipments.first.to_package }
+
+    before(:each) do
+      Spree::StockLocation.destroy_all
+      stock_location
+      order.create_proposed_shipments
+      order.shipments.count.should == 1
+      Spree::ActiveShipping::Config.set(:units => "imperial")
+      Spree::ActiveShipping::Config.set(:unit_multiplier => 1)
+      calculator.stub(:carrier).and_return(carrier)
+      Rails.cache.clear
+    end
+
+    describe "package.order" do
+      it{ expect(package.order).to eq(order) }
+      it{ expect(package.order.ship_address).to eq(address) }
+      it{ expect(package.order.ship_address.country.iso).to eq('US') }
+      it{ expect(package.stock_location).to eq(stock_location) }
+    end
+
+    describe "compute" do
```

Comment:
Same as above, we can define the subject for this block to be `subject { order.compute(package) }`.